index.js

```diff
@@ -1,6 +1,9 @@
 'use strict';
 
 const program = require('commander');
+
+const prettyjson = require('prettyjson');
```

Comment:
Please remove these newlines.

Also, not sure I think this is "pretty json" 🤔. I think the left-hand side should be displayed differently than the right. It'll make the relevant stuff "stick out" more.